precompile/assetsbridge/IAssetsBridge.sol

```diff
@@ -213,6 +213,35 @@ interface IAssetsBridge {
     * @notice Returns the minimum bridge-out amount (if set) for a Mezo token.
     * @param mezoToken The address of the token on the Mezo chain.
     * @return The current minimum amount for the token (0 if unset).
+    * @dev For bridge outs to the Bitcoin chain, the system will always check
+    *      the Bitcoin-specific minimum returned by getMinBridgeOutAmountForBitcoinChain
+    *      first, and fallback to the value returned by this function if the 
```

Comment:
I think this description needs updating.
If the target chain is Bitcoin and Bitcoin-specific is non-zero, we check both minimum bridge-out amounts.